app/src/main/java/edu/uc/kovaciad/slicetracker/dto/Artist.kt

```diff
@@ -10,9 +10,9 @@ import androidx.room.PrimaryKey
  * @param artistUrl: artist URL
  */
 @Entity(tableName = "Artist")
-class Artist (@PrimaryKey var aid: Int,
-    @ColumnInfo(name = "artistName")var artistName: String,
-    @ColumnInfo(name = "artistUrl")var artistUrl: String) {
+class Artist(@PrimaryKey var aid: Int,
+             @ColumnInfo(name = "artistName")var artistName: String,
+             @ColumnInfo(name = "artistUrl")var artistUrl: String) {
```

Comment:
For reformatting... 
1) A caution.  Significant reformats can make future commit diffs difficult, as git has to deal with significant reordering of the logic.
2) The IDE can do this automatically, and you can provide a template for the IDE to use.  Ensure all developers are using this same template, or the one that comes by default with the IDE, and have them reformat before committing.  This ameliorates issue 1, above.